src/components/SideNavigator/SideNavigator.tsx

```diff
@@ -280,13 +281,20 @@ export default () => {
             )
           }
 
+          controlComponents.unshift(
+            <ControlButton
+              key='syncButton'
```

Comment:
change this key to `${storage.id}-SettingsButton` and change line 277 to `${storage.id}-syncButton`